README.md

```diff
@@ -200,6 +137,78 @@ Travis uses a .travis.yml file in the root of your repository to learn about you
 |coverage_report|Enable [rspec-puppet coverage reports](https://rspec-puppet.com/documentation/coverage/). Defaults to `false`|
 |minimum_code_coverage_percentage|The desired code coverage percentage required for tests to pass. Defaults to `0`|
 
+## Configurations for Continuous Integration (CI) Environments
+
+The following keys allow you to configure and customize various third-party continuous integration (CI) environments for automatically testing your module.
+
+### Officially Supported CI Configurations
```

Comment:
IMO .... this doesn't really specify what "supported" means.

Reply:
Yeah, I mostly wanted to get this started, we can workshop how the two categories are actually described.